### PR TITLE
Exposure Notification Scaffolding 

### PR DESCRIPTION
**Pull request by octohub**
Merging the work from Rebecca to prevent merge conflicts in the future. Much of this is simply Google's sample exposure notification project. This is the foundation that the Google Apple Exposure Notification implementation on Android will be built off of.

### CODE REVIEW

**Review by johnschoeman — APPROVED** [review]
Hmm, a bit too much to review properly here, but I agree that its better to get this in sooner rather than later to avoid git merge issues.  I gave it a quick glance and looks fine to me.  Ideally wed be able to ship a bit more incrementally moving forward!

File: android/app/build.gradle
Comment (johnschoeman) [inline android/app/build.gradle:235]: 💯 does this mean that mauron85 only builds on gps!?

Comment (octohub) [reply android/app/build.gradle:235]: Yessir! I'll let @schmecs confirm as she wrote it

**Comment by octohub** [comment]
> Hmm, a bit too much to review properly here, but I agree that its better to get this in sooner rather than later to avoid git merge issues. I gave it a quick glance and looks fine to me. Ideally wed be able to ship a bit more incrementally moving forward!

Totally agree. This was just the initial handoff that I wanted to get in. I'd like to wait for @schmecs to review as well before merging.

File: android/app/build.gradle
Comment (schmecs) [reply android/app/build.gradle:235]: Yes! This can definitely be applied in some other places but this seemed like some low-hanging fruit.

File: android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/DebugMenuModule.java
Comment (schmecs) [inline android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/DebugMenuModule.java:46]: 👍

**Review by schmecs — APPROVED** [review]
Looks good to me, the failing test is from `develop`. @johnschoeman the overwhelming amount of new code is entirely my fault; I should have opened a couple of PRs when importing/hacking through the Google sample.

**Comment by octohub** [comment]
Thank you both, merging!

**Pull request merged**
